### ISSUE TITLE
Code completion proposals are not properly sorted

### ISSUE THREAD

**Issue by fbricon** [atom/ide-java#46]
When typing String, the completion proposals are pretty much irrelevant:
<img width="610" alt="screen shot 2017-11-16 at 9 38 59 am" src="https://user-images.githubusercontent.com/148698/32898813-3a1b38a2-cab7-11e7-83cf-64b0bb0f67b3.png">

Atom actually seems to ignore the sortText value of each proposal, which is supposed to translate the completion relevance into a proper sort:
```
        {
            "label": "String - java.lang",
            "kind": 7,
            "sortText": "999999164",
            "filterText": "String",
            "insertText": "String",
            "data": {
                "decl_signature": "Ljava.lang.String;",
                "pid": "74",
                "rid": "1",
                "uri": "file:/USERS/fbricon/Downloads/VSCode/src/Main.java"
            }
        },
```
For comparison, this is how VSCode displays the same results:
<img width="578" alt="screen shot 2017-11-16 at 10 23 47 am" src="https://user-images.githubusercontent.com/148698/32899188-46c027ce-cab8-11e7-9fd4-70274764611c.png">

**Comment by fbricon** [comment]
most likely related to https://github.com/atom/atom-languageclient/issues/133

**Comment by damieng** [comment]
Sorting was added to atom-languageclient in https://github.com/atom/atom-languageclient/commit/afef39bc5c1a551b425f78fe5e6bdac5cce266e7 should go out with the next ide-java release.

**Comment by damieng** [comment]
Fixed in ide-java 0.7.0

![image](https://user-images.githubusercontent.com/118951/33361178-ddcf16f2-d48b-11e7-8a86-1c7a7bf4a358.png)

**Comment by fbricon** [comment]
Nice! Now you just need to handle multiple textedits to get proper autocompletion, including auto-imports (#17)
